includes/Core/Assets/Assets.php

```diff
@@ -766,7 +766,19 @@ private function get_inline_user_data() {
 	 * @return array The dashboard sharing inline data to be output.
 	 */
 	private function get_inline_dashboard_sharing_data() {
-		$inline_data = array();
+		$all_roles   = wp_roles()->roles;
+		$inline_data = array( 'roles' => array() );
+
+		foreach ( $all_roles as $role => $details ) {
```

Comment:
Let's better use `$role_slug => $role_details` variables instead of just `$role` and `$details`.